waku/node/peer_manager/peer_manager.nim

```diff
@@ -128,9 +128,15 @@ proc addPeer*(pm: PeerManager, remotePeerInfo: RemotePeerInfo, origin = UnknownO
   if pm.peerStore[AddressBook][remotePeerInfo.peerId] == remotePeerInfo.addrs and
       pm.peerStore[KeyBook][remotePeerInfo.peerId] == remotePeerInfo.publicKey and
       pm.peerStore[ENRBook][remotePeerInfo.peerId].raw.len > 0:
-    trace "peer already managed and ENR info is already saved",
-      remote_peer_id = $remotePeerInfo.peerId
-    return
+    let incomingEnr = remotePeerInfo.enr.valueOr:
+      trace "peer already managed and incoming ENR is empty"
```

Comment:
I wonder if we could also print the `remote_peer_id` in this case too

Reply:
Great catch! 😍 Added it.
Thank you!